### DISCUSSION TITLE
Verify the supplier

### DISCUSSION THREAD

**Discussion by FabioArmonia** [fattureincloud/api#365] [Q&A] [answered]
Is there a way to verify a supplier when it is created through API? I would need something similar to the "VerifyEInvoiceXml" method but I could not find it inside the docs and the typescript sdk.

In alternative, is it a good idea to create a temporary invoice with that supplier and use the VerifyEInvoiceXml method on it? (I would delete the invoice soon after the verification)

**Comment by valmoz** [comment]
Hi @FabioArmonia what do you mean by "supplier verification"? What do you want to check?

Thanks
Mauro

**Reply by FabioArmonia** [reply]
In general I would like to be sure that if an invoice is created with that supplier it will not throw any error related to supplier validation failure (mainly vat code but also phone number, address, ...)

**Reply by valmoz** [reply, marked as answer]
OK, understood. We apply a formal check to the data you're sending (for example we verify the length of fields) but we don't check at this stage if the VAT code exists.

**Reply by FabioArmonia** [reply]
Ok I will apply a similar check before sending the data, thank you
